index.html

```diff
@@ -1,14 +1,459 @@
+            exchange,<br />
+            networking, and learning about Google developer technologies.
+          </p>
+          <div class="m-5 pb-5">
+            <p>
+              Join a local group and meet local developers with similar
+              interests.
+            </p>
+            <a class="btn-lg text-dark fw-semibold" href="#" role="button"
+              >Find a Google Developer Group near you ></a
+            >
+          </div>
+        </section>
+      </div>
+      <div class="d-flex justify-content-center border border-2">
+        <section class="text-center">
+          <div class="main-title position-relative m-5">
+            <h2 class="fw-semibold">Google Developer Group</h2>
+          </div>
+          <p class="d-block d-md-none white-bg m-5 p-4 ps-5 pe-5 text-wrap">
+            Google Developer Groups represents a network of technical
+            communities that offer opportunities for professional development
+            and connection.
+          </p>
+          <p class="d-none d-md-block white-bg m-5 p-4 ps-5 pe-5 text-wrap">
+            Google Developer Groups represents a network of technical
+            communities that offer <br />opportunities for professional
+            development and connection. Those who join can learn about the
+            latest Google<br />
+            tools and technologies, and can meet with local developers with
+            similar interests.
+          </p>
+          <div class="m-5">
+            <img
+              class="w-25 border border-2"
+              src="./visuals/gdg-member-badge.png"
+              alt=""
+            />
+          </div>
+        </section>
+      </div>
+      <div class="d-flex justify-content-center">
+        <section class="text-center">
+          <div class="main-title position-relative m-5">
+            <h2 class="fw-semibold">Earnable badges</h2>
+          </div>
+          <p class="white-bg mb-5 pb-4 ps-5 pe-5 text-wrap">
+            Earn these badges when you visit and register for 2022 DevFest
+            event.
+          </p>
+          <div
+            class="d-flex d-md-none m-5 flex-column justify-content-center align-items-center"
+          >
+            <img
+              class="w-50 m-1 border border-2"
+              src="./visuals/badge-visit.svg"
+              alt=""
+            />
+            <img
+              class="w-50 m-1 border border-2"
+              src="./visuals/badge-register.svg"
+              alt=""
+            />
+          </div>
+          <div class="d-none d-md-flex m-5 justify-content-center">
+            <img
+              class="w-50 m-1 border border-2"
+              src="./visuals/badge-visit.svg"
+              alt=""
+            />
+            <img
+              class="w-50 m-1 border border-2"
+              src="./visuals/badge-register.svg"
+              alt=""
+            />
+          </div>

```

Comment:
- [ ] The images in this section of the About Page should be an image in the background with text centred over the images. It would be nice to stick to the design requirement for this section. Kindly adjust to meet the design 🙏 

### Your Design
![image](https://user-images.githubusercontent.com/77106962/225115204-169dd795-de56-4f6c-9c4f-1e3e97072a62.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/225115323-06e42cd8-b33c-4b42-afc3-e4b6148fb8a9.png)